src/UserSpaceInstrumentation/include/UserSpaceInstrumentation/InstrumentProcess.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_INSTRUMENT_PROCESS_H_
+#define USER_SPACE_INSTRUMENTATION_INSTRUMENT_PROCESS_H_
+
+#include <absl/container/flat_hash_map.h>
+#include <absl/container/flat_hash_set.h>
+#include <sys/types.h>
+
+#include <cstdint>
+#include <memory>
+
+#include "OrbitBase/Result.h"
+#include "capture.pb.h"
+
+namespace orbit_user_space_instrumentation {
+
+class InstrumentedProcess;
+
+// `UserSpaceInstrumentation` is a globally unique object containing the bookkeeping for all user
+// space instrumentaion (in the `process_map_` member). It's lifetime is pretty much identical with
+// the lifetime of the profiling service.
+class UserSpaceInstrumentation {
```

Comment:
Nit. There is a header called `OrbitUserSpaceInstrumentation.h` and we have the `orbit_user_space_instrumentation` namespace. `UserSpaceInstrumentation` seems a bit generic and very close to the first two names, could `orbit_user_space_instrumentation::ProcessManager` or `orbit_user_space_instrumentation::InstrumentationManager` work?